### ISSUE TITLE
ErrorConstructor.captureStackTrace missing in TypeScript definitions

### ISSUE THREAD

**Issue by SyrupThinker** [denoland/deno#4702]
# Example
```typescript
// test.ts
const error = new Error("Example");
Error.captureStackTrace(error);
throw error;
```
## Expected `run` result
Successful typecheck and output of
```
error: Uncaught Error: Example
► file:///tmp/test.ts:2:7

2 Error.captureStackTrace(error);
        ^

    at file:///tmp/test.ts:2:7
```
(As if the above code were in a file named `test.js`)

## Actual `run` result
```
error TS2339: Property 'captureStackTrace' does not exist on type 'ErrorConstructor'.

► file:///tmp/test.ts:2:7

2 Error.captureStackTrace(error);
        ~~~~~~~~~~~~~~~~~
```

# System
```
> deno --version
deno 0.40.0
v8 8.2.308
typescript 3.8.3
```

**Comment by ry** [comment]
Just to be clear, it works in JS?

**Comment by SyrupThinker** [comment]
Yes, it works in JS.

**Comment by mudgen** [comment]
Someone told me that it was a typescript thing, that it doesn't work using typescript.

**Comment by kitsonk** [comment]
`captureStackTrace` is not a standard property, so that it why it isn't included in the standard TypeScript libraries.  It is perfectly fine to do in Deno, as it is a v8 thing, and we will always be off v8.

**Comment by nayeemrmn** [comment]
Oh, fixed in #5254.
